packages/pointer_interceptor/pointer_interceptor/example/lib/platforms/native_widget_ios.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// A widget representing an underlying platform view
+class NativeWidget extends StatelessWidget {
+  /// Constructor
+  const NativeWidget({super.key, required this.onClick});
+
+  /// Placeholder param to allow web example to work -
```

Comment:
the linter yells at me if I do this
https://dart.dev/tools/linter-rules/avoid_unused_constructor_parameters

Reply:
Ah, right, because it's matching the other version, but not inheriting from a base class. In this case it's fine to `// ignore:` that lint with a comment explaining why.